modules/ROOT/pages/tools/io-tool/filesending.adoc

```diff
@@ -0,0 +1,46 @@
+:imagesdir: 
+= File Sending
+
+When you share, you can choose to publish or send the files. Publishing means you send them out and anyone who is interested and has access can download them. On the other hand, when you send the files, you select a specific recipient.
+
+image::io-tool/sendorpublish.png[]
+
+Before sending the files you need to upload them to your _Data Storage_ (drag and drop, or click to choose). This is a pop up message you receive when uploading a valid data.
```

Comment:
Better "to send files via agrirouter"....
Better "you will be shown a popup after successful upload" and the "This is a ....." as a subtitle of an image